msal-go/advanced/managed-identity.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Managed identity with MSAL GO
+description: "How to use Azure managed identities in MSAL GO applications."
+---
+
+# Managed Identity with MSAL GO
+
+>[!NOTE]
+>This feature is available starting with [MSAL.GO](https://github.com/AzureAD/microsoft-authentication-library-for-go/releases/tag/v1.3.0-preview) version 1.3.0-preview
+
+A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. [Managed identities](/azure/active-directory/managed-identities-azure-resources/overview) in Azure eliminate the need for developers to handle these credentials manually. MSAL GO supports acquiring tokens through the managed identity service when used with applications running inside Azure infrastructure, such as:
+
+* [Azure VMs](https://azure.microsoft.com/free/virtual-machines/)
+* [Azure Arc](/azure/azure-arc/overview)
+
+For a complete list, refer to [Azure services that can use managed identities to access other services](/azure/active-directory/managed-identities-azure-resources/managed-identities-status).
+
+## Which SDK to use - Azure SDK or MSAL?
+
+MSAL libraries provide lower level APIs that are closer to the OAuth2 and OIDC protocols. 
+
+Both MSAL GO and [Azure SDK](/azure/developer/go/) allow to acquire tokens via managed identity. Internally, Azure SDK uses MSAL GO, and it provides a higher-level API via its `DefaultAzureCredential` and `ManagedIdentityCredential` abstractions.
+
+If your application already uses one of the SDKs, continue using the same SDK. Use Azure SDK, if you are writing a new application and plan to call other Azure resources, as this SDK provides a better developer experience by allowing the app to run on private developer machines where managed identity doesn't exist. Consider using MSAL if you need to call other downstream web APIs like Microsoft Graph or your own web API.
+
+## Creating Azure Resources
+
+You can create the resources needed for running the sample manually via the [Azure Portal](https://portal.azure.com/#home), or for a quick and
+concise breakdown of how to do it using Azure CLI or Azure Powershell, follow the instructions [here](/azure/developer/go/azure-sdk-authentication-managed-identity?tabs=azure-cli)
+
+## Quick start
+
+To quickly get started and see Azure Managed Identity in action, you can use one of the samples the team built for this purpose:
+
+> [Use Managed Identity sample](https://github.com/Azure-Samples/msal-managed-identity/tree/main/src/go)
+
+## Running Locally
```

Comment:
When you say running locally what does that mean? Managed identity scenarios does not work locally.